Dockerfile.ppc64le

```diff
@@ -95,25 +94,12 @@ RUN set -x \
 
 
 # Install Go
-# ppc64le doesn't have official go binaries, so use the version of go installed from the image
-# to build go from source.
-# NOTE: ppc64le has compatibility issues with older versions of go, so make sure the version >= 1.6
+# NOTE: official ppc64le go binaries weren't available until go 1.6.4 and 1.7.4
```

Comment:
Yeah, think it's fine like that. We can probably remove it after we move to the next version

> and I really don't want anyone to downgrade the version to say 1.7.3 and be confused as to why it doesn't work (this has happened a couple of times in the past)

I don't see a real reason for downgrading, and of course that's a risk of modifying the source of a project, but I'm ok with keeping the comment like it is now